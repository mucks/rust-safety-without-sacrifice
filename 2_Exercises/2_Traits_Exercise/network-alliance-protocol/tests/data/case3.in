# Allies count in report
STATE ForestCity 120
ALLY ForestCity Prospera
ALLY ForestCity Zuitzerland
CITIZEN ForestCity 1 Alice Builder
CITIZEN ForestCity 2 Akira Researcher
REPORT ForestCity
