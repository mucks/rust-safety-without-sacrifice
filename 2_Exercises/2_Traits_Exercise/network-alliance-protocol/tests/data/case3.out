REPORT ForestCity: pop=2 gdp=120 per_capita=60 allies=2
